certs/kubernetes.nix

```diff
@@ -16,6 +16,7 @@ let
       # Alternative names remain, as they might be useful for debugging purposes
       getAltNames "controlplane" ++
       getAltNames "loadbalancer" ++
+      lib.singleton "10.32.0.1" ++
```

Comment:
Nixpkgs standard library doesn't contain any functions for manipulating IPv4/6 addresses and CIDRs so I thing this being hardcoded is acceptable until a better solution can be found or such functions are added.